dubbo-cluster/src/test/java/org/apache/dubbo/rpc/cluster/loadbalance/ConsistentHashLoadBalanceTest.java

```diff
@@ -19,22 +19,34 @@
 import org.apache.dubbo.rpc.Invoker;
 
 import org.junit.Assert;
-import org.junit.Ignore;
 import org.junit.Test;
 
+import java.util.HashMap;
 import java.util.Map;
 import java.util.concurrent.atomic.AtomicLong;
 
+@SuppressWarnings("rawtypes")
 public class ConsistentHashLoadBalanceTest extends LoadBalanceBaseTest {
-    @Ignore
+    
     @Test
     public void testConsistentHashLoadBalance() {
-        int runs = 10000;
-        Map<Invoker, AtomicLong> counter = getInvokeCounter(runs, ConsistentHashLoadBalance.NAME);
-        for (Map.Entry<Invoker, AtomicLong> entry : counter.entrySet()) {
-            Long count = entry.getValue().get();
-            Assert.assertTrue("abs diff should < avg", Math.abs(count - runs / (0f + invokers.size())) < runs / (0f + invokers.size()));
-        }
+    	int runs = 10000;
+		long invokerCount0 = 0;
+		Map<Invoker, Long> invokerHited = new HashMap<>();
+		Map<Invoker, AtomicLong> counter = getInvokeCounter(runs, ConsistentHashLoadBalance.NAME);
+		for (Invoker minvoker : counter.keySet()) {
+			Long count = counter.get(minvoker).get();
+
+			if (count == 0) {
+				invokerCount0 ++;
+			} else {
+				invokerHited.put(minvoker, count);
+			}
+		}
+
+		Assert.assertEquals(counter.size() - 1, invokerCount0);
```

Comment:
Is it ok to put assert failure message, so that in future if any one go through with UT code or UT report it can help him on addition of code.
e.g. 
        Assert.assertEquals("Same consistent hashed call should go to same node only",counter.size() , untouchedNodesCount);
        Assert.assertEquals("Should call only same node for one consistent hash",1, touchedNodesHitMap.size());
        Assert.assertEquals("Total call should me made to same one node",runs, touchedNodesHitMap.values().iterator().next().intValue());